src/patternfly/components/Masthead/examples/masthead.md

```diff
@@ -0,0 +1,204 @@
+---
+id: 'Masthead'
+beta: true
+section: components
+cssPrefix: pf-c-masthead
+---
+
+## Examples
+### Basic
+```hbs
+{{#> masthead masthead--id="basic-masthead"}}
+  {{> masthead-toggle}}
+  {{#> masthead-main}}
+    {{#> masthead-brand}}
+      Logo
+    {{/masthead-brand}}
+  {{/masthead-main}}
+  {{> masthead-content}}
+{{/masthead}}
+```
+
+### Basic with mixed content
+```hbs
+{{#> masthead masthead--id="basic-masthead-with-mixed-content"}}
+  {{> masthead-toggle}}
+  {{#> masthead-main}}
+    {{#> masthead-brand}}
+      Logo
+    {{/masthead-brand}}
+  {{/masthead-main}}
+  {{#> masthead-content}}
+    {{#> l-flex}}
+      <span>Testing text color</span>
+      {{#> button button--modifier="pf-m-primary"}}
+        testing
+      {{/button}}
+      {{#> l-flex-item l-flex-item--modifier="pf-m-align-flex-end"}}
+        {{#> button button--modifier="pf-m-primary"}}
+          testing
+        {{/button}}
+      {{/l-flex-item}}
+    {{/l-flex}}
+  {{/masthead-content}}
+{{/masthead}}
+```
+
+### Basic with horizontal nav
+```hbs
+{{#> masthead masthead--id="basic-masthead-with-horizontal-nav"}}
+  {{> masthead-toggle}}
+  {{#> masthead-main}}
+    {{#> masthead-brand}}
+      Logo
+    {{/masthead-brand}}
+  {{/masthead-main}}
+  {{#> masthead-content}}
+    <nav class="pf-c-nav pf-m-horizontal pf-m-scrollable" aria-label="Global">
```

Comment:
just a note, we'll probably want to remove this, or if we keep it, use HBS. My vote would be to remove, and we'll show it in a page/nav demo.